ddd/all_words.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'word'
+
+wolke = Word.new('die', 'Wolke')
+sonne = Word.new('die', 'Sonne')
+katze = Word.new('die', 'Katze')
+kind = Word.new('das', 'Kind')
+wasser = Word.new('das', 'Wasser')
+stern = Word.new('der', 'Stern')
+ofen = Word.new('der', 'Ofen')
+
+@all_words = Word.all
```

Comment:
It's not really clear what's going on with this instance variable: It gets assigned in one file and is used in another. And it doesn't belong to a class.
Instead, in the other class, I'd just use `Word.all` directly.

Reply:
ok, change it to `word = Word.all.sample` in main file